### DISCUSSION TITLE
Trying to add new variable in sei_growth.py

### DISCUSSION THREAD

**Discussion by loveinSH** [pybamm-team/PyBaMM#3498] [Q&A]
Hello, I am trying to add a new variable to tune the value of j_sei.
So I added it like this in submodel part sei_growth.py

    def get_coupled_variables(self, variables):
        phase_param = self.phase_param
        # delta_phi = phi_s - phi_e
        T = variables["Negative electrode temperature"]
        j_sei_test = variables["j_sei_test"]

Meanwhile I have defined j_sei_test in lithium-ion-parameters like this.
self.j_sei_test = pybamm.Variable('j_sei_test')

But once I added j_sei_test, it will raise a error:
pybamm.expression_tree.exceptions.ModelError: Missing variable for submodel 'negative primary active material': 'X-averaged SEI interfacial current density'.
Check the selected submodels provide all of the required variables.

I am not sure what is the problem now.

**Comment by valentinsulzer** [comment]
when you create a `Variable` it needs to be added to `model.rhs` or `model.algebraic`, and also to `model.initial_conditions`.
If you just want `j_sei_test` to be a fixed value you should make it a `Parameter`.
